.github/workflows/ci.yml

```diff
@@ -10,13 +10,13 @@ env:
   # plus all the README.md files of the affected packages.
   RUST_MIN_VER: "1.85"
   # List of packages that can not target Wasm.
-  NO_WASM_PKGS: "--exclude masonry --exclude xilem"
+  NO_WASM_PKGS: "--exclude masonry --exclude masonry_winit --exclude xilem"
   # Only some of our examples support Android (primarily due to extra required boilerplate).
   ANDROID_TARGETS: "-p xilem --example mason_android --example calc_android --example stopwatch_android --example variable_clock_android --example http_cats_android --example to_do_mvc_android"
   # Whether the workspace contains Rust code using the debug_assertions configuration option.
   USING_DEBUG_ASSERTIONS: "true"
   # The files stored in LFS the tests need to access
-  LFS_FILES: "masonry/src/**/screenshots/*.png"
+  LFS_FILES: "masonry/**/screenshots/*.png masonry_core/**/screenshots/*.png"
```

Comment:
Hmm. I'm surprised you managed to make this work. When we had to do the same thing in Vello, I had to make the changes in https://github.com/linebender/vello/pull/837

Reply:
Well I'm not sure it *is* working, so I might need to copy your changes.

Reply:
Yup, this was what was causing the CI failures.

The fix was to use a single glob for all crates, as discussed in #911.